### PR TITLE
switch dom order, add aria-labelledby

### PR DESCRIPTION
**Pull request by morethanreal**
For #8. @notwaldorf PTAL cc @cdata

### CODE REVIEW

**Comment by morethanreal** [comment]
lgtm

File: paper-toolbar.html [hunk 1]
```diff
@@ -240,6 +252,10 @@
 
       is: 'paper-toolbar',
 
+      hostAttributes: {
+        'role': 'toolbar'
```
Comment (notwaldorf) [inline paper-toolbar.html:256]: does it need a `tabindex="0"` as well ?

Comment (morethanreal) [inline paper-toolbar.html:256]: It's not interactive, what does it mean for it to have `tabindex=0`?

Comment (notwaldorf) [inline paper-toolbar.html:256]: Oh, blerg, i was thinking it's 'paper-tabs`. All good!

File: paper-toolbar.html [hunk 2]
```diff
@@ -285,6 +301,51 @@
...
+        var contents = Polymer.dom(this.root).querySelectorAll('content');
+        for (var content, index = 0; content = contents[index]; index++) {
+          var nodes = Polymer.dom(content).getDistributedNodes();
+          for (var node, jndex = 0; node = nodes[jndex]; jndex++) {
+            if (node.hasAttribute && node.hasAttribute('title')) {
+              if (node.id) {
+                labelledBy.push(node.id);
+              } else {
+                var id = 'paper-toolbar-label-' + Math.floor(Math.random() * 10000);
+                console.log(id);
```
Comment (notwaldorf) [inline paper-toolbar.html]: console.log leftover?

File: paper-toolbar.html [hunk 3]
```diff
@@ -285,6 +301,51 @@
...
+        var labelledBy = [];
+        var contents = Polymer.dom(this.root).querySelectorAll('content');
+        for (var content, index = 0; content = contents[index]; index++) {
+          var nodes = Polymer.dom(content).getDistributedNodes();
+          for (var node, jndex = 0; node = nodes[jndex]; jndex++) {
+            if (node.hasAttribute && node.hasAttribute('title')) {
+              if (node.id) {
+                labelledBy.push(node.id);
+              } else {
+                var id = 'paper-toolbar-label-' + Math.floor(Math.random() * 10000);
```
Comment (notwaldorf) [inline paper-toolbar.html:336]: Hmm, what happens in the unlikely case we get a duplicate? we just miss reading one, right? which i think is maybe ok?

**Comment by notwaldorf** [comment]
lgtm % the two comments

File: paper-toolbar.html (hunk 3 above)
Comment (morethanreal) [inline paper-toolbar.html:336]: It'll pick up the first one in the DOM if it's a dup.

**Pull request merged**